bfjit.c

```diff
@@ -133,8 +130,15 @@ bool interpret(Ops ops)
     return result;
 }
 
+struct JitMemoryData {
```

Comment:
The reason as to which this is not a `typedef struct` is because I wanted to forward declare it inside of the `update_buffer` signature, else GCC produced a warning.